vtex/faststore
# feat: Add fuzzy parameter in search query

## What's the purpose of this pull request?

Adds the possibility to pass the fuzzy parameter in the search query

## How to test it?

Change the parameter value between 0, 1 or auto and check the search result return

![image](https://github.com/vtex/faststore/assets/39604283/93c1bba5-5845-445c-91ff-da5083afe596)
![image](https://github.com/vtex/faststore/assets/39604283/f5d38480-2bac-475d-93a5-a783d70d762e)


I used the carrefour account to test since that's where I found the problem, searching for the term 'Molico' was bringing up products like 'Molho de tomate'
